tests/immer/atomWithImmer.test.tsx

```diff
@@ -1,15 +1,15 @@
-import React from 'react'
 import { fireEvent, render } from '@testing-library/react'
 import { useAtom } from '../../src/index'
 import { atomWithImmer } from '../../src/immer'
 import { getTestProvider } from '../testUtils'
+import { FC } from 'react'
```

Comment:
Please move this to L1.

Reply:
You can consider the [order](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md) rule from eslint-plugin-import. It provides auto-fixing. Do you want to configure it in the project?

Reply:
Sound good. Let's address in a separate PR. I might have some preference on the config. (But, I would like to see your suggestion too.)